internal/signalfx-agent/pkg/monitors/collectd/disk/disk.go

```diff
@@ -38,5 +38,5 @@ type Monitor struct {
 
 // Configure configures and runs the plugin in collectd
 func (m *Monitor) Configure(conf *Config) error {
-	return m.SetConfigurationAndRun(conf)
+	return m.SetConfigurationAndRun(conf, collectd.WithDeprecationWarningLog("disk-io"))
```

Comment:
@atoulme - can we provide an alternative? Would the host metrics receiver do the deed, for instance?